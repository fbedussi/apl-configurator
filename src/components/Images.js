import React from 'react';

const Images = ({ sources }) => {
    if (!(sources && sources.length) ) {
        return null;
    }

    return <div class="imagesContainer">
            sources.map(src => <img className="image" src={`images/${src}`}/>)
        </div>
}

export default Images;