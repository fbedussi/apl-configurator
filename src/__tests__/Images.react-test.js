import React from 'react';
import Images from '../components/Images';
import renderer from 'react-test-renderer';

const props = {
  sources: ['src1', 'src2']
};

test('Images component', () => {
  const component = renderer.create(
    <Images {...props} />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});