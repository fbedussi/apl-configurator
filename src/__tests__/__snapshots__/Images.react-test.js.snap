// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Images component 1`] = `
<div
  className="imagesContainer"
>
  <img
    className="image"
    src="images/src1.jpg"
  />
  <img
    className="image"
    src="images/src2.jpg"
  />
</div>
`;
